modules/machineset-yaml-nutanix.adoc

```diff
@@ -0,0 +1,137 @@
+
+ifeval::["{context}" == "creating-infrastructure-machinesets"]
+:infra:
+endif::[]
+
+:_content-type: REFERENCE
+[id="machineset-yaml-nutanix_{context}"]
+= Sample YAML for a machine set custom resource on Nutanix
+
+This sample YAML defines a Nutanix machine set that creates nodes that are labeled with
+ifndef::infra[`node-role.kubernetes.io/<role>: ""`.]
+ifdef::infra[`node-role.kubernetes.io/infra: ""`.]
+
+In this sample, `<infrastructure_id>` is the infrastructure ID label that is based on the cluster ID that you set when you provisioned the cluster, and
+ifndef::infra[`<role>`]
+ifdef::infra[`<infra>`]
+is the node label to add.
+
+[source,yaml]
+----
+apiVersion: machine.openshift.io/v1beta1
+kind: MachineSet
+metadata:
+  labels:
+    machine.openshift.io/cluster-api-cluster: <infrastructure_id> <1>
+ifndef::infra[]
+    machine.openshift.io/cluster-api-machine-role: <role> <2>
+    machine.openshift.io/cluster-api-machine-type: <role> <2>
+  name: <infrastructure_id>-<role>-<zone> <3>
+endif::infra[]
+ifdef::infra[]
+    machine.openshift.io/cluster-api-machine-role: <infra> <2>
+    machine.openshift.io/cluster-api-machine-type: <infra> <2>
+  name: <infrastructure_id>-<infra>-<zone> <3>
+endif::infra[]
+  namespace: openshift-machine-api
+  annotations: <4>
+    machine.openshift.io/memoryMb: "16384"
+    machine.openshift.io/vCPU: "4"
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      machine.openshift.io/cluster-api-cluster: <infrastructure_id> <1>
+ifndef::infra[]
+      machine.openshift.io/cluster-api-machineset: <infrastructure_id>-<role>-<zone> <3>
+endif::infra[]
+ifdef::infra[]
+      machine.openshift.io/cluster-api-machineset: <infrastructure_id>-<infra>-<zone> <3>
+endif::infra[]
+  template:
+    metadata:
+      labels:
+        machine.openshift.io/cluster-api-cluster: <infrastructure_id> <1>
+ifndef::infra[]
+        machine.openshift.io/cluster-api-machine-role: <role> <2>
+        machine.openshift.io/cluster-api-machine-type: <role> <2>
+        machine.openshift.io/cluster-api-machineset: <infrastructure_id>-<role>-<zone> <3>
+endif::infra[]
+ifdef::infra[]
+        machine.openshift.io/cluster-api-machine-role: <infra> <2>
+        machine.openshift.io/cluster-api-machine-type: <infra> <2>
+        machine.openshift.io/cluster-api-machineset: <infrastructure_id>-<infra>-<zone> <3>
+endif::infra[]
+    spec:
+      metadata:
+        labels:
+ifndef::infra[]
+          node-role.kubernetes.io/<role>: ""
+endif::infra[]
+ifdef::infra[]
+          node-role.kubernetes.io/infra: ""
+endif::infra[]
+      providerSpec:
+        value:
+          apiVersion: machine.openshift.io/v1
+          cluster:
+            type: uuid
+            uuid: <cluster_uuid>
+          credentialsSecret:
+            name: nutanix-creds-secret
+          image:
+            name: <infrastructure_id>-rhcos <5>
+            type: name
+          instanceType: <instance_type> <6>
```

Comment:
i don't think `instanceType` is utilized, looking through the [api definition](https://github.com/openshift/api/blob/master/machine/v1/types_nutanixprovider.go#L15-L67) i'm not seeing it. is there another source for this field?